src/frontend/src/components/public/PIAListTable/index.tsx

```diff
@@ -52,9 +52,16 @@ const PIAListTable = ({ headings, pias, sorting }: IDataTable) => {
           {pias.map((pia, index) => (
             <tr key={index}>
               <td
+                // tabIndex={0}
                 role="button"
                 className="fw-bold"
                 onClick={() => onTitleClick(pia.id)}
+                onKeyDown={(event) => {
+                  if (event.code === 'Enter' || event.code === 'Space') {
```

Comment:
I think we only need Enter. That's how other accessibility buttons are behaving at the moment.

Reply:
I will remove the space code